socorro/external/es/mappings/correlations_index_settings.json

```diff
@@ -4,9 +4,6 @@
             "_all": {
                 "enabled": false
             },
-            "_source": {
-                "compress": true
-            },
```

Comment:
Yes. It's not just that we can, it's that the option is not accepted anymore, so we have to remove it. :)